dbt/adapters/databricks/impl.py

```diff
@@ -429,10 +429,20 @@ def parse_columns_from_information(  # type: ignore[override]
         return columns
 
     def get_catalog(
-        self, manifest: Manifest, selected_nodes: Optional[Set] = None
+        self, manifest: Manifest, selected_nodes: Optional[Set[Any]] = None
+    ) -> Tuple[Table, List[Exception]]:
+        if selected_nodes:
+            relations: Set[BaseRelation] = {
+                self.Relation.create_from(self.config, n) for n in selected_nodes
+            }
+        else:
+            relations = set(self._get_catalog_relations(manifest))
+        return self.get_catalog_by_relations(manifest, relations)
+
+    def get_catalog_by_relations(
+        self, manifest: Manifest, catalog_relations: Set[BaseRelation]
```

Comment:
Minor nit, we call this `relations` in `dbt-core`, not `catalog_relations`. That technically changes the signature.

Reply:
Will correct this, thanks.  Python is loosy goosy.